src/lua/LuaJson.cpp

```diff
@@ -34,10 +38,8 @@ static void _push_json_to_lua(lua_State *l, Json &obj)
 	case Json::value_t::array: {
 		lua_newtable(l);
 		size_t size = obj.size();
-		lua_pushinteger(l, size);
-		lua_setfield(l, -2, "n");
 		for (size_t idx = 0; idx < size; idx++) {
-			lua_pushinteger(l, idx);
+			lua_pushinteger(l, idx + 1);
```

Comment:
Thanks for catching this - I hadn't actually exercised this codepath until writing the equipment rework.